labkey-client-api/README.md

```diff
@@ -0,0 +1,69 @@
+# The LabKey Remote API Library for Java
+
+Welcome to the LabKey Remote API library for Java. This library allows Java 
```

Comment:
nit: As welcomed as I feel I don't think this verbiage is necessary.

```suggestion
This library allows Java 
```

Reply:
Yeah, it seemed odd to me too.  Gone now.